dist/css/main.css

```diff
@@ -0,0 +1,259 @@
+body {
+  background: #000;
+  font-family: 'monospace', sans-serif;
+  font-weight: 400;
+  position: relative;
+}
+
+a, h1, h2, h3, h4, h5, span {
+  color: #fff;
+}
+
+.heading {
+  font-size: 22px;
+  font-weight: 500;
+}
+
+.subheading {
+  font-size: 16px;
+  padding-bottom: 60px;
+}
+
+p {
+  color: #fff;
+  font-weight: 400;
+  font-size: 16px;
+  letter-spacing: 0.4px;
+  line-height: 28px;
+}
+
+hr {
+	border-top: 2px solid #fff;
+	width: 120px;
+}
+
+#menu, #gallery, #team {
+	padding-top: 100px;
+	padding-bottom: 100px;
+}
+#menu, #contact {
+  background: #000;
+}
+
+.parallax-section {
+  background-attachment: fixed !important;
+  background-size: cover !important;
+}
+
+/* start navigation */

```

Comment:
It’s better to delete commented code. If you need to review a previous version of your code, use git history, your code is clear enough to understand 🙂 ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

Kindly remove all in this file.